collection+xattrs.go

```diff
@@ -521,7 +522,7 @@ func (c *Collection) writeWithXattr(
 			}
 			if xattrKey == "_sync" {
 				// The "_sync" xattr has two properties that get macro-expanded:
-				e.expandSyncXattrMacros(parsedXattr)
+				e.expandSyncXattrMacros(parsedXattr, c.bucket.uuid, sgBucketOpts)
```

Comment:
It sounds as though SG will be using macro substitutions in other xattrs, like the VV one, so the above test `if xattrKey == "_sync"` should probably be changed to `if strings.HasPrefix(xattrKey, "_")`